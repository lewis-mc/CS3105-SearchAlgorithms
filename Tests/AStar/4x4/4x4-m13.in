4 4 1 3 4 8 6 2 11 7 5 0 10 12 9 13 14 15