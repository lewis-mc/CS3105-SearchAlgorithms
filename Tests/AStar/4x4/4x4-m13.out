1 13 10 11 7 8 4 3 2 6 5 9 13 14 15